api/src/main/kotlin/io/spine/protodata/protobuf/MessageClass.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2025, TeamDev. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.protodata.protobuf
+
+import com.google.protobuf.Descriptors.Descriptor
+import com.google.protobuf.Message
+import kotlin.reflect.KClass
+
+/**
+ * Returns a Protobuf descriptor for this message [Class].
+ */
+public fun Class<out Message>.getDescriptor(): Descriptor =
+    getDeclaredMethod("getDescriptor").invoke(null) as Descriptor
+
+/**
+ * Returns a Protobuf descriptor for this message [KClass].
+ */
+public fun KClass<out Message>.getDescriptor(): Descriptor = java.getDescriptor()
```

Comment:
They also [come](https://github.com/SpineEventEngine/validation/pull/192/files#diff-ec2ae3a3f10ff63a7085eccf7a47cbda13c38a13da75b258647d3f37857c09caR51) from `validation`. But they look too generic for the `validation` library.

These are also [similar](https://github.com/SpineEventEngine/validation/pull/192/files#diff-49793842ebfeaf401f8cd6048d0d4a64b9edbaa9bfb40cb0f35920ceafe2ae04R35) extensions to return `Message.Builder`, but they are tests-only. As for now, I can't say how they can be used in `main` sources.